components/UVCStream/UVCStream/UVCStream.cpp

```diff
@@ -78,7 +83,8 @@ static void UVCStreamHelpers::camera_stop_cb(void *cb_ctx)
 
 static uvc_fb_t *UVCStreamHelpers::camera_fb_get_cb(void *cb_ctx)
 {
-  (void)cb_ctx;
+  auto *mgr = static_cast<UVCStreamManager *>(cb_ctx);
+  (void)mgr;
```

Comment:
that `(void)cb_ctx` is there only to tell the compiler not to emit warnings since it's "used". If you're needing the stream manger here, you can safely remove the `(void)mgr`